auction-house/js/src/AuctionHouseProgram.ts

```diff
@@ -116,8 +120,8 @@ export class AuctionHouseProgram extends Program {
         auctionHouse.toBuffer(),
         treasuryMint.toBuffer(),
         tokenMint.toBuffer(),
-        Buffer.from(price, 'utf8'),
-        Buffer.from(tokenSize, 'utf8'),
+        new BN(price).toArrayLike(Buffer, 'le', 8),
+        new BN(tokenSize).toArrayLike(Buffer, 'le', 8),
```

Comment:
Normally you'd call bn.toBuffer() but is not available in browsers. Instead use toArrayLike to support browser.

It was incorrect before with strings being used. The seeds are computed based on u8 in the program so needs to be a number in js.